storage-access-api/helpers.js

```diff
@@ -150,10 +150,32 @@ async function CanFrameWriteCookies(frame, keep_after_writing = false) {
   return can_write;
 }
 
+// Sets a cookie in an unpartitioned context by creating a new frame
+// and requesting storage access in the frame.
+async function SetUnpartitionedCookie(origin) {
+  let frame = await CreateFrame(`${origin}/storage-access-api/resources/script-with-cookie-header.py?script=embedded_responder.js`);
```

Comment:
This actually is meant to set a cookie in a top-level context for the given origin, not add CORS headers specifically. This is so we can for access (or no access) to this "unpartitioned" cookie when validating storage access, rather than the presence of the document.cookie variable which is not specified to disappear without storage access by the Storage Access API.

Reply:
I guess my question would've been better phrased as "why can't we reuse `create_cookie` here?" AFAICT the main difference between `create_cookie` and the handler for this URL is that this handler will attach the CORS headers, and `create_cookie` won't.

(There's also the `alert` in `create_cookie`, but I agree with Johann, that can probably be removed.)

If the CORS headers in the handler aren't important, can we remove them, just to simplify the scenario being tested?

Reply:
Removing the CORS headers to simplify.